src/main/java/mate/academy/springbootstore/controller/AuthenticationController.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springbootstore.controller;
+
+import jakarta.validation.*;
+import lombok.*;
+import mate.academy.springbootstore.dto.*;
+import mate.academy.springbootstore.service.*;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/auth")

```

Comment:
add `server.servlet.context-path=/api` to `application.properties`
```suggestion
@RequestMapping("/auth")
```